task/04-arrays-tasks.js

```diff
@@ -286,7 +321,8 @@ function propagateItemsByPositionIndex(arr) {
  *   [ 10, 10, 10, 10 ] => [ 10, 10, 10 ]
  */
 function get3TopItems(arr) {
-   throw new Error('Not implemented');
+   let reversed=arr.reverse()
+   return reversed.slice(0,3)
```

Comment:
If test cases weren't sorted , this solution wouldn't work.
Add sort method
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

spaces